Adds Tomcat

bendem opened the pull request:
Added tomcat. Let me know if there is any information missing or malformed.

bendem committed:
Proposition for tomcat

bendem committed:
Rename tomcat.yml to tomcat.md

usta commented on products/tomcat.md:
```diff
@@ -0,0 +1,47 @@
+    release: 2003-09-06
+    eol: 2012-09-30
+    latest: "5.5.36"
+  - 
```
try to remove this extra `-`

usta replied:
@bendem could you fix this one too , thank you

captn3m0 commented on products/tomcat.md:
```diff
@@ -0,0 +1,47 @@
+---
+> The Apache Tomcat® software is an open source implementation of the [Jakarta Servlet](https://projects.eclipse.org/projects/ee4j.servlet), [Jakarta Server Pages](https://projects.eclipse.org/projects/ee4j.jsp), [Jakarta Expression Language](https://projects.eclipse.org/projects/ee4j.el), [Jakarta WebSocket](https://projects.eclipse.org/projects/ee4j.websocket), [Jakarta Annotations](https://pro ...
+
+The update and release strategy is _complex_. For release policy information, see <https://tomcat.apache.org/whichversion.html>.
```
Complexity of the policy is not a good reason for us to omit it entirely. While upstream projects might have to cover everything, we can reduce complexity in a lot of ways - we don't need to cover historical policies for EoL releases, and we only cover stable releases.

@bendem Wanna give this a shot? Or I can try in a bit.

captn3m0 committed:
Update tomcat.md

Simplified description, and removed empty release

captn3m0 committed:
Update tomcat.md

- Adds a Java compatibility matrix
- Adds note about stability
- Adds lifecycle guidance

captn3m0 committed:
[tomcat] Fix links

captn3m0 committed:
[tomcat] Fix logo image

captn3m0 reviewed (approved):
Should be good to merge

BiNZGi committed:
Update 8.5.77

BiNZGi reviewed (commented):
Verified all the details and updated 8.5.77, all okay -> merge
Thanks for the contribution @bendem :100: looking for more from you...
